### ISSUE TITLE
Some missing files and requirements for training

### ISSUE THREAD

**Issue by kentegrate** [chentuochao/Sound_Bubble#1]
Hello!

I am grateful for sharing your impressive work! While attempting to reproduce the results, I encountered a few issues that I'd like to bring to your attention:

1. **Missing Requirements:**
   The `pyroomacoustics==0.7.7` requirement should be specified in the `requirements2.txt` file. I noticed that the API for `pyroomacoustics` has undergone significant changes from version 0.8.0 onwards, leading to compatibility issues when installing the latest version using `pip install pyroomacoustics`.

2. **Missing Modules:**
   The script `generate_adaptive_dataset.py` attempts to import a module named `generate_realdata_from_denoised` which appears to be absent from the repository. Could this module be added?

3. **Access Issues:**
   - I am interested in obtaining real-world datasets and checkpoints. However, the provided link contains a prompt to "Please fill this form," which is currently non-functional as it cannot be clicked or interacted with.
   - Additionally, the hyperlink leading to [dryad](https://doi.org/10.5061/dryad.r7sqv9smv) seems broken. I would like the link to be updated.

Thank you again for sharing your great work!

Best regards,
Ken

**Comment by chentuochao** [comment]
Hi, Thanks for reaching out!
1. I think newer pyroomacoustics also works
2. uploaded! Sorry for mistakes.
3. dryad dataset link is ready now, you re-access again.

Let me know whether you have more questions!

**Comment by kentegrate** [comment]
Hi!

First of all, thank you so much for your assistance! I'm pleased to inform you that with the generated data on my side, training the model was successful.

However, I did come across a couple of issues that I wanted to address:

1. **Import issue**
   I noticed that in several places within the code, there are imports from `sep.helpers.utils`, as seen here: [search results](https://github.com/search?q=repo%3Achentuochao%2FSound_Bubble%20sep.&type=code). However, it seems that this module is missing. If 'sep' refers to the package name, perhaps additional steps are required to ensure proper import functionality.

2. **TypeError with `read_audio_file_torch()`**
   There seems to be a mismatch between the function definition and invocation of `read_audio_file_torch()`. The [function definition](https://github.com/chentuochao/Sound_Bubble/blob/5d8e4c028c25f522d48887073c5131a283dca577/helpers/utils.py#L100) suggests it takes only one argument, but in the [code usage](https://github.com/chentuochao/Sound_Bubble/blob/5d8e4c028c25f522d48887073c5131a283dca577/src/datasets/general_multisrc_dataset_dis_embed.py#L136), two arguments are given. It appears that the downsampling ratio, which should be accounted for, is currently missing from the function parameters.
Would it be possible to get the latest version of the code that resolves these issues? In reviewing the configuration files, I see that the downsampling ratio is set to 1 across all cases, so one option would be just to remove the downsampling ratio argument to replicate the results. Nevertheless, having the updated code would be extremely helpful.

Thanks once again for your support!

Best regards,
Ken
